framework/src/timesteppers/FunctionDT.C

```diff
@@ -163,9 +161,8 @@ FunctionDT::computeDT()
   if (local_dt < _min_dt)
     local_dt = _min_dt;
 
-  if (_cutback_occurred && (local_dt > _dt * _growth_factor))
+  if ((local_dt > (_dt * _growth_factor)) && _dt > 0)
```

Comment:
If we want this to only effect cut time steps and not limit the rate timestep increase by functions
``` 
if (_cutback_occurred && (local_dt > _dt * _growth_factor))
  local_dt = _dt * _growth_factor;
else {
  _cutback_occurred = false;
}
```